source/beamlines.md

```diff
@@ -36,6 +37,28 @@ The value of `zero_point` is the name of a `line item` that marks the reference
 To make things unambiguous, the reference `item` must have zero length.
 In most cases, this means that the `zero_point` cannot be a `BeamLine`.
 
+Setting optional `periodic` Boolean to `True` indicates that the `BeamLine` is something like a 
+storage ring where the particle beam recirculates through the `BeamLine` multiple times.
+Setting `periodic` to `False` is used to indicate that the `BeamLine` is something like a 
+Linac or any other line that is "single pass". 
+
+Notice that a setting `periodic` to `True` does **not** mean that the downstream end of
+the last element of the `BeamLine` has the same [floor](#s:floor) coordinates as the floor
+coordinates at the beginning. Setting `periodic` to `True` simply signals to a program that
+it may be appropriate to calculate closed (periodic) orbits and Twiss parameters
+as opposed to calculating orbits and Twiss
+parameters based upon orbit and Twiss parameters set by the User for the beginning of the `BeamLine`.  
+Indeed, it is sometimes convenient to treat `BeamLines` as periodic even though there is no 
```

Comment:
I know that we had quite a bit of discussion on this already at the meeting, but this makes me feel like 'periodic' may not be the best qualifier here. What about "closed" or "open"?

Reply:
`closed` and `open`  will be removed.